### PR TITLE
bug: fix the misc se-build error

### PR DESCRIPTION
**Pull request by huoqifeng**
Fixes: #1712

### CODE REVIEW

**Review by liudalibj — APPROVED** [review]
LGTM

File: ibmcloud/image/build.sh [hunk 1]
```diff
@@ -113,7 +113,8 @@ fi
 
 printf "\nCopying partitions from the base image $src_img_path\n"
 
-qemu-img create -f qcow2 -b "$base_img_path" "$src_img_path" $disksize
+cp -f "$base_img_path" "$src_img_path"
```
Comment (liudalibj) [inline ibmcloud/image/build.sh:116]: Is https://github.com/confidential-containers/cloud-api-adaptor/blob/main/podvm/Makefile#L76 need be updated too?

Comment (huoqifeng) [reply ibmcloud/image/build.sh:116]: @liudalibj the problem is `-b "$base_img_path"`, I think it's OK for code in Makefile.

**Review by genjuro214 — APPROVED** [review]
LGTM

**Pull request merged**